kv/helpers.go

```diff
@@ -53,10 +53,3 @@ func Each(ctx context.Context, tx Tx, fnc func(k Key, v Value) error, opts ...It
 	}
 	return it.Err()
 }
-
-// CreateBucket is a helper to create buckets upfront without writing any key-value pairs to it.
-func CreateBucket(_ context.Context, tx Tx, key Key) error {
```

Comment:
@dennwc The reason was because there's no need to create keys with nil values (buckets). Removing this as well as the version number (constant at 2) reduced the number of read/writes significantly in my use case. 

Will drop it for this PR since it's not suitable for your upstream repo.